data/app.metainfo.xml

```diff
@@ -35,6 +35,17 @@
   </screenshots>
   <content_rating type="oars-1.1" />
   <releases>
+    <release version="1.5" date="2024-10-xx">
+      <description translatable="no">
+        <ul>
+          <li>Update to GNOME 47</li>
```

Comment:
Maybe we can also include the change from #115?

Reply:
happy to help

Reply:
Sure!